chartfx-math/src/main/java/de/gsi/math/MathGenBase.java

```diff
@@ -4098,8 +3018,9 @@ public static short RMS(short[] data, int length) {
         val1 *= norm;
         val2 *= norm;
         // un-biased rms!
-        return (short) Math.sqrt(Math.abs(val2 - val1 * val1));
+        return sqrt(java.lang.Math.abs(val2 - val1 * val1)); //// codegen: returncast all
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'java.lang.Math.abs' due to existing implicit import 'java.lang.*'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)